external_samples/rev_touch_sensor.py

```diff
@@ -0,0 +1,53 @@
+from component import Component, PortType, InvalidPortException, EmptyCallable
+
+class RevTouchSensor(Component):
+    def __init__(self, ports : list[tuple[PortType, int]]):
+        self.is_pressed = None
```

Comment:
It seems like is_pressed should be a boolean, so the value should be False instead of None.

Reply:
It can't be, because when the system boots up you don't know if the touch sensor is currently pressed or not.   If you start out saying it isn't, then you'll get an immediate callback for pressed even though that isn't accurate.   So you have to start out before your first read with something that knows it hasn't read hardware yet.

Leaving this comment open to see if you agree or not.

Reply:
OK. That makes sense.